httpdocs/js/webots-cloud.js

```diff
@@ -164,28 +178,24 @@ document.addEventListener('DOMContentLoaded', function() {
       const words = data.url.substring(19).split('/');
       const repository = `https://github.com/${words[0]}/${words[1]}`;
       const animation = `https://${words[0]}.github.io/${words[1]}/${words[3]}`;
+      const title = data.title === '' ? '<i>anonymous</i>' : data.title;
       const updated = data.updated.replace(' ',
         `<br><i class="is-clickable fas fa-sync" id="sync-${data.id}" data-url="${data.url}" title="Re-synchronize now"></i> `
       );
-      const title = data.title === '' ? '<i>anonymous</i>' : data.title;
-      let type;
-      let competitors;
-      if (data.type == 'demo') {
-        type = `<i class="fas fa-grin-stars fa-lg" title="${data.type}"></i>`;
-        competitors = '';
-      } else {
-        const icon = (data.type == 'benchmark') ? 'award' : 'trophy';
-        type = `<i class="fas fa-${icon} fa-lg" title="${data.type}"></i>`;
-        const url = data.url.substring(0, data.url.lastIndexOf("/worlds/"));
-        competitors = `<a class="has-text-dark" href="${url}/competitors.txt" target="_blank">${data.competitors}</a>`;
-      }
+      let icon = 'question';
+      if (data.type === 'demo')
+        icon = 'chalkboard-teacher';
+      else if (data.type === 'benchmark')
+        icon = 'award';
+      else if (data.type === 'competition')
+        icon = 'trophy';
```

Comment:
Double initialization:
```suggestion
      let icon;
      if (data.type === 'demo')
        icon = 'chalkboard-teacher';
      else if (data.type === 'benchmark')
        icon = 'award';
      else if (data.type === 'competition')
        icon = 'trophy';
      else
        icon = 'question';
```